systers_portal/meetup/migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
All the migrations except for this one, don't make sense in the current context. So please remove them.